twistr/score.go

```diff
@@ -0,0 +1,126 @@
+package twistr
+
+type ScoreLevel int
+
+const (
+	Nothing ScoreLevel = iota
+	Presence
+	Domination
+	Control
+)
+
+const WIN int = -1
+
+func VPAward(level ScoreLevel, r Region) int {
```

Comment:
We actually don't need to do that — SEAsia scoring can be implemented directly in its own function, because it is not a recurring event and it isn't re-scored in final scoring.